phpstan.neon.dist

```diff
@@ -14,3 +14,5 @@ parameters:
     checkMissingVarTagTypehint: true
     checkMissingTypehints: true
     checkUninitializedProperties: true
+    scanFiles:
+        - vendor/symfony/dependency-injection/Loader/Configurator/ContainerConfigurator.php
```

Comment:
Why is this needed ? Vendor is generally always scanned isn't it ?

Reply:
https://github.com/phpstan/phpstan-symfony#configuration

If I don't add this I can't use `service`, `param` or `abstract_arg` since phpstan reports them as not existent symbols.